libs/coin-modules/coin-stacks/src/config.ts

```diff
@@ -0,0 +1,27 @@
+import { CurrencyConfig } from "@ledgerhq/coin-framework/config";
+import "@ledgerhq/types-cryptoassets";
```

Comment:
I think you don't need `import "@ledgerhq/types-cryptoassets"`

Reply:
done in 180d1120ab48e336b28446bf59ac02b894f34975